spec/factories/employees.rb

```diff
@@ -2,5 +2,11 @@
   factory :employee do
     sequence(:email) { |n| "employee#{n}@corpname.com.br" }
     password { '123456' }
+
+    trait :with_shipping_co do
+      after(:build) do |employee|
+        employee.shipping_co = FactoryBot.build(:shipping_co)
+      end
```

Comment:
```suggestion
      shipping_co
```

Acho que o método original do factory_bot faria examente isso

Reply:
Eu estava com duvidas de como fazer, encontrei uns exemplos assim no google, mas acho que era um jeito antigo de fazer, esse ficou bem mais simples.